{"level":"error","message":"Something failed miserably! Something failed miserably!","stack":"Error: Something failed miserably!\n    at Object.<anonymous> (/home/zaxiya/node programs/first-app/11 Handling and Logging Errors/vidly/index.js:58:26)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"Something failed miserably! Something failed miserably!","stack":"Error: Something failed miserably!\n    at Object.<anonymous> (/home/zaxiya/node programs/first-app/11 Handling and Logging Errors/vidly/index.js:59:26)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"Something failed miserably! Something failed miserably!","stack":"Error: Something failed miserably!\n    at Object.<anonymous> (/home/zaxiya/node programs/first-app/11 Handling and Logging Errors/vidly/index.js:61:26)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at node:internal/main/run_main_module:17:47"}
